pkg/remote/google/repository/asset.go

```diff
@@ -37,17 +39,19 @@ func NewAssetRepository(client *asset.Client, config config.GCPTerraformConfig,
 }
 
 func (s assetRepository) searchAllResources(ty string) ([]*assetpb.ResourceSearchResult, error) {
+	cacheKey := "googleSearchAllResources"
```

Comment:
You're right, I just though it wasn't worth another PR. But I can split anyway if you prefer.

To explain this change further: it's important that we use unique identifiers for keys in the cache so it doesn't conflict with other repositories.

Reply:
> To explain this change further: it's important that we use unique identifiers for keys in the cache so it doesn't conflict with other repositories.

For now we use a cache instance per provider, so there is no risk of collisions with AWS or Azure.

Reply:
> For now we use a cache instance per provider, so there is no risk of collisions with AWS or Azure.

I meant other repositories within the same provider. I just looked at AWS repositories and we use the service name as prefix to avoid conflicts. Here we simply use the asset API so you're right let's keep it that way for now.